### ISSUE TITLE
Ember Template Lint throwing weird errors in CI

### ISSUE THREAD

**Issue by Alonski** [ember-template-lint/ember-template-lint#777]
In `ember-file-upload` Travis has started to fail for me because of Template Lint false negatives.

A failing Job: https://travis-ci.org/adopted-ember-addons/ember-file-upload/jobs/559624710

I don't understand exactly why this is happening. Locally everything is linted correctly with the same dependencies.

The errors:
```sh
$ yarn lint:hbs
yarn run v1.17.3
$ ember-template-lint .
/home/travis/build/adopted-ember-addons/ember-file-upload/addon/components/file-dropzone/template.hbs
  -:-  error  Cannot read property 'MaybeLocal' of undefined  undefined
/home/travis/build/adopted-ember-addons/ember-file-upload/addon/components/file-upload/template.hbs
  -:-  error  Cannot read property 'OpenElement' of undefined  undefined
/home/travis/build/adopted-ember-addons/ember-file-upload/tests/dummy/app/templates/application.hbs
  -:-  error  Cannot read property 'Unknown' of undefined  undefined
/home/travis/build/adopted-ember-addons/ember-file-upload/tests/dummy/app/templates/docs.hbs
  -:-  error  Cannot read property 'Text' of undefined  undefined
/home/travis/build/adopted-ember-addons/ember-file-upload/tests/dummy/app/templates/index.hbs
  -:-  error  Cannot read property 'Text' of undefined  undefined
/home/travis/build/adopted-ember-addons/ember-file-upload/tests/dummy/app/templates/not-found.hbs
  -:-  error  Cannot read property 'Text' of undefined  undefined
```

**Comment by alexdiliberto** [comment]
Seeing issues as well here

```
$ ember-template-lint .
/home/travis/build/alexdiliberto/ember-transformicons/addon/templates/components/t-add.hbs
  -:-  error  Cannot read property 'OpenElement' of undefined  undefined
/home/travis/build/alexdiliberto/ember-transformicons/addon/templates/components/t-form.hbs
  -:-  error  Cannot read property 'OpenElement' of undefined  undefined
/home/travis/build/alexdiliberto/ember-transformicons/addon/templates/components/t-grid.hbs
  -:-  error  Cannot read property 'OpenElement' of undefined  undefined
/home/travis/build/alexdiliberto/ember-transformicons/addon/templates/components/t-loader.hbs
  -:-  error  Cannot read property 'OpenElement' of undefined  undefined
```

```
dummy/templates/application.hbs
  -:-  error  Cannot read property 'Unknown' of undefined  undefined
dummy/templates/docs.hbs
  -:-  error  Cannot read property 'Text' of undefined  undefined
dummy/templates/index.hbs
  -:-  error  Cannot read property 'Text' of undefined  undefined
dummy/templates/not-found.hbs
  -:-  error  Cannot read property 'Text' of undefined  undefined
===== 4 Template Linting Errors
```

**Comment by Alonski** [comment]
Thanks @alexdiliberto 
I used Yarn Resolutions to force the version to ^1.4.0 in package.json

```json
  "resolutions": {
    "ember-cli-template-lint/**/ember-template-lint": "^1.4.0"
  }
```
